components/Flipper/Flipper.tsx

```diff
@@ -563,14 +631,18 @@ function Flipper(props: Props) {
                     {isLoggedIn ? (
                         ''
                     ) : (
-                        <span>
-                            These are flips that were previosly found (~5 min ago). Anyone can use these and there is no cap on estimated profit. Keep in mind
-                            that these are delayed to protect our paying supporters. If you want more recent flips purchase our{' '}
-                            <a target="_blank" rel="noreferrer" href="/premium">
-                                premium plan.
-                            </a>
-                        </span>
+                        <>
```

Comment:
what the heck is that

Reply:
Thats a Fragment (https://reactjs.org/docs/fragments.html#short-syntax)
It's a nice way to group multiple elements in one component without having to group it into a useless div